src/shared/interfaces.ts

```diff
@@ -0,0 +1,9 @@
+export interface MovieData {
+  id: string;
+  title: string;
+  releaseDate: number;
+  imageUrl: string;
+  description: string;
+}
+
+export type Genre = 'Crime' | 'Documentary' | 'Horror' | 'Comedy';
```

Comment:
Try to use enums and use all enum values when rendering buttons.

Hint: If you struggle with how to get all values of enum, make an enum in TS repl and see what is the output in the JS

Reply:
done. thx for tip